support/provenance/terraform/provider.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  version = "~> 3.0"
+  version = ">= 4.7"
```

Comment:
@jimmie per discussions we have been having with @nutjob4life about issues we have encountered with python version collisions, do we want to pin to a more specific version?

Reply:
@tloubrieu-jpl ☝️

Reply:
I've just learned how that works and I agree with what Sean said